### PR TITLE
Invgen 44613 multi sheets

### PR DESCRIPTION
**Pull request by Houmicek**

### CODE REVIEW

File: WebApplication/ClientApp/src/components/forgePdfViewExtension.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// *******************************************
+// My Color Extension
```
Comment (erymski) [inline WebApplication/ClientApp/src/components/forgePdfViewExtension.js:2]: Leftover? :)
Please add the default header

File: WebApplication/ClientApp/src/components/forgePdfViewExtension.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+ForgePdfViewExtension.prototype.createUI = function() {
+    if (this.initialized === true)
+        return;
+
+    this.initialized = true;
+    const viewer = this.viewer;
+    const urn = viewer.getState().seedURN;
+    const numPages = viewer.model.loader.pdf.numPages;
+    let actualPage = 1;
```
Comment (erymski) [inline WebApplication/ClientApp/src/components/forgePdfViewExtension.js:43]: To be sure: index of the first page is `1`, not `0`?

**Review by obergrd — APPROVED** [review]

File: AppBundles/ExportDrawingAsPdfPlugin/Automation.cs [hunk 3]
```diff
@@ -74,7 +74,8 @@ public void RunWithArguments(Document doc, NameValueMap map)
                     LogTrace("Drawing saved");
                     var pdfPath = System.IO.Path.Combine(dir, "Drawing.pdf");
                     LogTrace("Exporting drawing to: " + pdfPath);
-                    drawingDocument.SaveAs(pdfPath, true);
+                    ExportIDWToPDF(drawingDocument, pdfPath);
```
Comment (erymski) [inline AppBundles/ExportDrawingAsPdfPlugin/Automation.cs:77]: Please increase version here
https://github.com/Developer-Autodesk/forge-configurator-inventor/blob/05b415b0a619ce84b864bb3688df37f83702771b/AppBundles/ExportDrawingAsPdfPlugin/Properties/AssemblyInfo.cs#L9-L10

**Review by erymski — CHANGES_REQUESTED** [review]
Need to increase version number on appbundle change

File: WebApplication/ClientApp/src/components/forgePdfViewExtension.js (hunk 1 above)
Comment (obergrd) [reply WebApplication/ClientApp/src/components/forgePdfViewExtension.js:2]: I will update it

File: WebApplication/ClientApp/src/components/forgePdfViewExtension.js (hunk 2 above)
Comment (obergrd) [reply WebApplication/ClientApp/src/components/forgePdfViewExtension.js:43]: yes, default pageNumber is 1

**Review by erymski — APPROVED** [review]

**Pull request merged**